src/components/CardGrid/CardGrid.stories.jsx

```diff
@@ -6,11 +6,25 @@ export default {
   title: 'Components/CardGrid',
   component: CardGrid,
   parameters: {
+    docs: {
+      source: {
+        // https://github.com/storybookjs/storybook/issues/12747
+        type: 'code',
+      },
+    },
     layout: 'fullscreen',
   },
 };
 
-const Template = (args) => <CardGrid {...args} />;
+const Template = ({ cards, ...args }) => (
+  <CardGrid
+    {...args}
+    cards={cards.map(({ actionButtonText, ...card }) => ({
+      ...card,
+      actionButton: <Button>{actionButtonText}</Button>,
```

Comment:
This appears to have made the button required. i.e. you can no longer render a CardGrid without buttons. See the default story for an example.